Agent/AgentWar3/UnitRefList/Ability/AbilityInterfaced/CAbilityHero.h

```diff
@@ -59,7 +59,7 @@ class CAbilityHero : CAbilityInterfaced {
     int field_12C;
     int field_130;
     int field_134;
-    int field_138;
```

Comment:
It's based on the "Owning player", so if it's player 2's first hero the slot value will be 1, if it's his second it's 2, but red might then make his first and it will still be 0.

Generally this might also be the identifier for the hero's place in order overall (since hero 1 is always in front of hero 2 in the selected units and similar).